index.html

```diff
@@ -6,14 +6,100 @@
+                            src="../imgs/Whildlife/Landscape/photo-1456926631375-92c8ce872def.jpeg" alt=""></div>
+                    <div class="col-6 sm-col-6 md-col-4 lg-col-4 xl-col-3"><img
+                            src="../imgs/Whildlife/Landscape/wrev1ljvQ6KlfyljCQG0_lion.jpg" alt=""></div>
+                    <div class="col-6 sm-col-6 md-col-4 lg-col-4 xl-col-3"><img
+                            src="../imgs/Whildlife/Landscape/photo-1570422593863-bd38ef7ce050.jpeg" alt=""></div>
+                    <div class="col-6 sm-col-6 md-col-4 lg-col-4 xl-col-3"><img
+                            src="../imgs/Whildlife/Landscape/photo-1502051897604-58025075ce54.jpeg" alt=""></div>
+                    <div class="col-6 sm-col-6 md-col-4 lg-col-4 xl-col-3"><img
+                            src="../imgs/Whildlife/Landscape/photo-1474511320723-9a56873867b5.jpeg" alt=""></div>
+                    <div class="col-6 sm-col-6 md-col-4 lg-col-4 xl-col-3"><img
+                            src="../imgs/Whildlife/Landscape/photo-1517993037474-692208825419.jpeg" alt=""></div>
+                    <div class="col-6 sm-col-6 md-col-4 lg-col-4 xl-col-3"><img
+                            src="../imgs/Whildlife/Landscape/photo-1557973254-3b9af1ee03aa.jpeg" alt=""></div>
+                    <div class="col-6 sm-col-6 md-col-4 lg-col-4 xl-col-3"><img
+                            src="../imgs/Whildlife/Landscape/photo-1526653662218-3ab1e4d6ec2a.jpeg" alt=""></div>
+                    <div class="col-6 sm-col-6 md-col-4 lg-col-4 xl-col-3"><img
+                            src="../imgs/Whildlife/Landscape/photo-1486688680290-be46662593bd.jpeg" alt=""></div>
+                    <div class="col-6 sm-col-6 md-col-4 lg-col-4 xl-col-3"><img
+                            src="../imgs/Whildlife/Landscape/photo-1460999158988-6f0380f81f4d.jpeg" alt=""></div>
+                    <div class="col-6 sm-col-6 md-col-4 lg-col-4 xl-col-3"><img
+                            src="../imgs/Whildlife/Landscape/photo-1551716540-3ee7beb24f85.jpeg" alt=""></div>
+                </div>
+            </section>
+        </div>
+        <section class="second-bg-img">
+        </section>
+        
+        <section id="about" class="about container col-6">
+            <h1>About.</h1>
+            <span>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Quo sapiente eius cumque pariatur eveniet,
+                ad excepturi ratione dignissimos repellendus temporibus.</span>
+        </section>
+
+        <section class="third-bg-img">
+        </section>
+
+        <section id="contact" class="contact">
+            <h1>Contact.</h1>
+            <div class="container">
+                <form class="row">
+                    <label class="col-10 sm-col-10 md-col-6 lg-col-5 info-label" for="name">Enter Your Name</label>
+                    <input class="col-10 sm-col-10 md-col-6 lg-col-5" type="text" id="name" name="name" value="Name" required><br>
```

Comment:
I recommend using placeholder= rather than value= in inputs

Reply:
Thank you for the note Alessandro :) Noted and done!